index.bs

```diff
@@ -3981,12 +3981,19 @@ the iterator objects returned by <code class="idl">entries</code>,
 <code class="idl">keys</code>, <code class="idl">values</code>, and {{@@iterator}} are
 actual [=array iterator objects=].
 
-Interfaces with iterable declarations must not
-have any [=interface members=]
-named "<code>entries</code>", "<code>forEach</code>",
-"<code>keys</code>", or "<code>values</code>",
-or have any [=inherited interfaces=]
-that have [=members=] with these names.
+Interfaces with iterable declarations must not have any [=regular attributes=] or [=constants=]
+named
+"<code>entries</code>",
+"<code>forEach</code>",
+"<code>keys</code>", or
+"<code>values</code>",
+or have any [=inherited interfaces=] that have [=regular operations=], [=regular attributes=] or
+[=constants=] with these names.
+
+Note: Operations with these names are allowed on interfaces with iterable declarations and will
```

Comment:
Why this change?  Regular operations with these names didn't use to be allowed, and they still shouldn't be, imo.